docs/source/gbd2020_models/risk_exposures/wasting_state_exposure/index.rst

```diff
@@ -490,7 +490,66 @@ pool.
 Here we include equations for the transition probabilities, and in the section 
 that follows we will detail how to calculate all the variables used
 
-.. list-table:: Wasting transition probability equations
+.. list-table:: NEW WASTING TRANSITION PROBABILITY EQNS
+   :widths: 5 15 10 10
+   :header-rows: 1
+
```

Comment:
Do you define the `ap` terms anywhere? I don't see it, but I could be missing it.

Reply:
yes! they're defined here: 
![image](https://user-images.githubusercontent.com/57412727/125343162-9c83e600-e30a-11eb-90c0-c657dbbb016d.png)

Reply:
(the table that starts on line 593 @rmudambi )

Reply:
To be consistent with the current wasting treatment PR, 
duration of cat 1 = 54.5 days; 
duration of cat 2 = 52.4 days (0.512*63 + 0.488*41.3)
duration of cat 3 = ? (can leave as 365 for now)

I know Abie wants to simplify things so we might revise this after discussions with Abie.